source/includes/stackpath/_delivery_rules.md

```diff
@@ -49,6 +49,91 @@ Attributes | &nbsp;
 `stackId`<br/>*string* | The ID of the stack that the delivery rule belongs to.
 `siteId`<br/>*string* | The ID of the site that the delivery rule is applied to.
 
+<!-------------------- CREATE A DELIVERY RULE -------------------->
+
+### Create a delivery rule
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliveryrules?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+
+> Request body example for creating a delivery rule:
+
+```js
+{
+ "name": "rule",
+  "deliveryRuleConditions": [
+  {
+    "trigger": "HEADER",
+    "operator": "MATCHES",
+    "target": "my-header",
+    "httpMethods": ["GET"]
+  }],
+  "deliveryRuleActions": [
+    {
+      "actionType": "REDIRECT",
+      "redirectUrl": "https://my-url.com"
+    }]
+}
+```
+
+> The above commands return a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliveryrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Delivery Rules which allow you to modify CDN settings based on a number of elements like URL, HTTP headers, and more.
+
+Query Params | &nbsp;
+---- | -----------
+`siteId`<br/>*UUID* | The ID of the site for which to create the delivery rule. This parameter is required.
+
+Required| &nbsp;
+------------------------| -----------
+`name`<br/>*string* | The name of the delivery rule.
+`deliveryRuleConditions`<br/>*Array[[Condition](#stackpath-condition)]* | At least one condition.
+`deliveryRuleActions`<br/>*Array[[Action](#stackpath-action)]* | At least one action.
+
+Optional| &nbsp;
+------------------------| -----------
+`slug`<br/>*string* | A delivery rule's programmatic name.
+
+#### Condition
+
+Required| &nbsp;
+------------------------| -----------
+`trigger`<br/>*Enum* | Trigger for the condition. Possible values: `COOKIE`, `HEADER`, `HTTP_METHOD`, `STATUS_CODE`, and `URL`.
+`operator`<br/>*Enum* | Operator for the condition. Possible values: `EQUALS`, `MATCHES`, and `MATCHES_REGULAR_EXPRESSION`.
+`target`<br/>*String* | The target of the condition.
+`httpMethods`<br/>*Array[Enum]* | HTTP methods for the condition. Possible values are: `GET`, `POST`, `PUT`, `DELETE`, `HEAD`, `PATCH`, and `OPTIONS`.
```

Comment:
Worth mentioning that this field is only used when the `HTTP_METHOD` trigger is used.

Reply:
Fixed in db21586ef5b052e4398519b44633184ed6ae92da